src/components/KeyValueTable.tsx

```diff
@@ -45,7 +46,9 @@ const KeyValueTable: React.FC<KeyValueTableProps> = ({ header, data }: KeyValueT
           {data?.map((kv) => (
             <AlternatingRow key={kv?.k}>
               <TableCell className={classes.tableCell}>{kv?.k}</TableCell>
-              <TableCell className={classes.tableCell}>{kv?.v}</TableCell>
+              <TableCell className={classes.tableCell}>
+                <TruncateText text={kv?.v ?? ''} />
```

Comment:
@wooneh this isn't where I want the truncation applied - sorry. For the K/V I think it best to show the whole thing as there's no easy way to see the full payload.

Where it **should** be used is the file_name column of the new FileList component. And it IS! just checked :).